k8s/device-monitor/stage-device-monitor-api.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     matchLabels:
       app: sta-device-monitor
   replicas: 2
+  revisionHistoryLimit: 3
```

Comment:
Just curious, what does revisionHistoryLimit help with?  @123MwanjeMike

Reply:
The revision history of 3 allows us to roll back to an older version of a deployment should we choose to.
The default is 10 which can keep a long list of unwanted replica sets